Data/Primitive/Types.hs

```diff
@@ -120,15 +140,26 @@ class Prim a where
     -> State# s
     -> State# s
 
+  {-# MINIMAL (sizeOfType | sizeOf#), (alignmentOfType | alignment#), indexByteArray#, readByteArray#, writeByteArray#, setByteArray#,
+    indexOffAddr#, readOffAddr#, writeOffAddr#, setOffAddr# #-}
+
+-- | A dummy value of type @a@.
+dummy :: a
+dummy = errorWithoutStackTrace "dummy"
```

Comment:
Should this be marked noinline? cc @andrewthad

Reply:
Looking at the core, it isn't inlined, but it shouldn't hurt to explicitly mark it as noinline.

Reply:
You'd think GHC would automatically mark bottoming things `NOINLINE`, but I guess not?